docs/intro/our-calls.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Our Calls
+description: A list of they types of calls, meetings, work sessions, and jams we
+  have in our internal community.
+---
+In SourceCred we frequently work asynchronously, but we also make use of voice and/or video meetings when we need to get coordinated. There are a few different types of calls we have in our community, below you'll find a list of our regular calls and their expectations. 
+
+All calls/meetings take place in the voice channels of the SourceCred Discord server.
+
+You can see what types of calls are currently scheduled by looking at our \[calendar](http://sourcecred.io/calendar).
+
+# Weekly Community Call
+
+Audience: Anyone who is interested in SourceCred, especially newcomers
+
+What: The founder, and some members of the core team host a call for any newcomers or community members to attend, introduce themselves, ask questions, etc.
```

Comment:
Can we just make it "the core team"? That way if I miss some meetings, we're not defaulting on this promise.